src/SprykerEco/Zed/PunchoutCatalogs/PunchoutCatalogsDependencyProvider.php

```diff
@@ -40,9 +47,13 @@ public function provideCommunicationLayerDependencies(Container $container): Con
         parent::provideCommunicationLayerDependencies($container);
         $container = $this->addUtilDateTimeService($container);
         $container = $this->addPunchoutCatalogConnectionPropelQuery($container);
+        $container = $this->addPunchoutCatalogConnectionSetupPropelQuery($container);
+        $container = $this->addPunchoutCatalogConnectionCartPropelQuery($container);
         $container = $this->addPunchoutCatalogTransactionPropelQuery($container);
         $container = $this->addCompanyBusinessUnitFacade($container);
+        $container = $this->addCompanyUserFacade($container);
         $container = $this->addConnectionFormatPlugins($container);
```

Comment:
And now we have another issue:
- https://github.com/spryker-eco/punchout-catalogs/pull/7/files?file-filters%5B%5D=.php&file-filters%5B%5D=.twig&file-filters%5B%5D=.xml#diff-b4f0311c6869ac28b0550b7574a081c2R26

- https://github.com/spryker-eco/punchout-catalogs/pull/7/files?file-filters%5B%5D=.php&file-filters%5B%5D=.twig&file-filters%5B%5D=.xml#diff-b4f0311c6869ac28b0550b7574a081c2R31

Naming mismatch. If you added `PunchoutCatalogs` prefix to plugin stack related methods - you should add it everywhere (to keep methods naming match).

So here, on line 55 you have `addConnectionFormatPlugins` method, but since it was added earlier, I think, you should completely get rid of those prefix, that you've added earlier (to not change existing stuff).

Reply:
Reverted